lib/nodes.py

```diff
@@ -63,6 +80,8 @@ def import_nodeinfo(nodes, nodeinfos, now, assume_online=False):
         node['flags']['online'] = False
         node['flags']['gateway'] = False
 
+        mark_uplinks(nodeinfo, node)
```

Comment:
why isn't mark_fastd_uplinks(nodeinfo, node) called directly?
if there might be other types of uplinks in the future, one could still add the layer.